content/zmc-challenges/first-website/your-first-html-page/_index.md

```diff
@@ -34,61 +34,61 @@ You can also experiment a bit by writing HTML code and seeing what it does. Expe
 
 Please note that `index.html` is NOT the same as `Index.html`. Capital letters matter a lot in code. Please be extra careful about naming your files with the right case!
 
 ### 2. Let's make some content 
 
-You'll be adding a bunch of content to your html file. Please make very sure that you are meeting all the requirements below:
+You'll be adding a bunch of content to your html file. Please make sure that you are meeting all the requirements below:
 
 ### 2.1 Add a `<head>` element
 
 You might not know what a `head` tag does just yet. You can learn about it [here](https://www.w3schools.com/html/html_head.asp).
 
-Write some html code that meets the following requirements:
+Write some HTML code that meets the following requirements:
 
 - Your page must make use of a `<head>` element 
 - There must be one `<title>` element inside the head. Experiment a bit and make sure you know what the title does. You can give your page whatever title text you want. Eg: `<title>First website</title>`. 
 - There must be one `<meta>` element that sets the charset to `UTF-8`. It should look like this `<meta charset="UTF-8">`. 
 
 #### 2.2 The `<body>`
 
-Write some html code that meets the following requirements:
+Write some HTML code that meets the following requirements:
 
 - it needs to have ONLY ONE `<h1>` heading 
 - it needs to have ONE OR MORE `<h2>` headings
-- it needs to have ONE OR MORE paragraphs
-- there should be no text that is sitting outside of a heading or paragraph element.
+- It needs to have ONE OR MORE paragraphs
+- There should be no text that is sitting outside of a heading or paragraph element.
 
 #### 2.3 Clean and valid code
 
 Please make sure that your code is clean and valid. Here are a few things to look out for:
 
-- the document type declaration must be correct. The document type declaration is the very first line in your html file
-- you should be making use of valid tags - pay attention to case (all tags are lowercase)
-- any tag you open should get closed
-- your index.html file should meet ALL of the requirements described
-- you should be able to open your index.html file in a browser and it should behave sensibly
-- Make use of an online HTML validator to check your own work. Here is a validator: https://www.freeformatter.com/html-validator.html 
+- The document type declaration must be correct. The document type declaration is the very first line in your html file
+- You should be making use of valid tags - pay attention to case (all tags are lowercase)
+- Any tag you open should get closed
+- Your index.html file should meet ALL of the requirements described
+- You should be able to open your index.html file in a browser and it should behave sensibly
+- Make use of an online HTML validator to check your own work. Here is a validator: https://validator.w3.org/ and [here[(https://www.youtube.com/watch?v=XXecps3jEVw) is a video to show you how to make use of the validator.
```

Comment:
The video link is not linked correctly:
![image](https://github.com/Umuzi-org/ACN-syllabus/assets/54069197/a9481b70-74cc-4ea3-b92e-0f121443d492)